client/deis.py

```diff
@@ -1348,6 +1335,24 @@ def _read_config_from_path(self, path):
 
         return env_dict
 
+    def _display_configuration(self, configuration, oneline=False, highlight=[]):
```

Comment:
let's make the `oneline` and `highlight` options as [keyword arguments](http://stackoverflow.com/a/1419160) to make this a little easier to maintain.